example/src/cat/cats.service.spec.ts

```diff
@@ -0,0 +1,78 @@
+// Inspired by https://github.com/jmcdo29/testing-nestjs/blob/master/apps/mongo-sample/src/cat/cat.service.spec.ts
+
+import { Cat } from "./cat.model";
+import { CatsService } from "./cats.service";
+import { ReturnModelType } from "@typegoose/typegoose";
+import { TestingModule, Test } from "@nestjs/testing";
+import { getModelToken } from "nestjs-typegoose";
+
+const mockCat: (mock?: {
```

Comment:
i dont think this will work without the right types

Reply:
@hasezoey I refactored tests with Mongo memory server, what do you think?

Reply:
didnt run it yet, but definitely looks better (and cleaner than before)